Source/KerbalConstructionTime/Utilities/Utilities.cs

```diff
@@ -973,6 +1011,16 @@ public static BuildListVessel AddVesselToBuildList(BuildListVessel blv)
             return protoTechNodes;
         }
 
+        public static double GetShipProgress(BuildListVessel ship, out double origBP, out double buildTime, out double difference, out double progress)
```

Comment:
Hmm, that got a lot messier than I originally thought. Maybe return new BP amount, original completion % and new completion % ?